playwright/features/pubSource.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Specific Pub Sources
```

Comment:
This is cucumber test scenarios. Basically all these steps are being tested and test steps are dynamically written in our test folder. Cucumber is used for easier documentation and quick test writing.

Reply:
Ok thanks! I understand. 

These test scenarios are brittle and verbose.

I recommend considering breaking out the specific functionality to test: 
- Search functionality - ensure search results show up as expected given a very generic query
- Reader functionality - ensure a specific file/type or source is readable.

Reply:
Since we've added Playwright & Cucumber to the app, I've learned a lot more about how to structure Playwright tests, and the general best practice is to use a [page object model](https://playwright.dev/docs/pom). As an example, you can see how we are doing this in [pjr](https://github.com/NYPL/ereading-clients/tree/main/apps/pjr/playwright/discover): each page has its own folder that contains a spec file and a corresponding page file.

Also, I'd like to re-evaluate if we need to use Cucumber at all. Imo the layer of abstraction it provides doesn't provide any value and adds unnecessary complexity.

My recommendation would be to remove Cucumber and begin refactoring our tests to use the page object model.